mirrord/cli/src/config.rs

```diff
@@ -798,6 +799,10 @@ pub(super) struct ListTargetArgs {
     /// Specify config file to use.
     #[arg(short = 'f', long, value_hint = ValueHint::FilePath)]
     pub config_file: Option<PathBuf>,
+
+    /// Specify the type of target to be retrieved. If `None`, all types are retrieved.
+    #[arg(short = 't', long)]
+    pub target_type: Option<Vec<TargetType>>,
```

Comment:
Does the help command show how to specify multiple targets? Like is it `mirrord ls --target_type "deployments;pods"`? Can you add this as docs for this field?